report_xlsx_pdf/tests/test_report.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Openindustry.it (<https://openindustry.it>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo.tests import common
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from xlrd import open_workbook
+except ImportError:
+    _logger.debug("Can not import xlrd`.")
+
+
+class TestReport(common.TransactionCase):
+    def setUp(self):
+        super(TestReport, self).setUp()
+        report_object = self.env["ir.actions.report"]
+        self.xlsx_report = self.env["report.report_xlsx.abstract"].with_context(
+            active_model="res.partner"
+        )
+        self.report_name = "report_xlsx.partner_xlsx"
+        self.report_xlsx_to_pdf = True
```

Comment:
I don't understand this. It is never used... You never tested that is is transformed. This test is for report_xlsx standard. Can you test the code from this module?

Reply:
self.report_xlsx_to_pdf = True
is the new parameter for this app

Reply:
But you are editing self, in this case, the TestClass, it does not make sense. You should do `self.xlsx_report.report_xlsx_to_pdf = True'

Reply:
@andreampiovesana can you attend this comment? Thanks!